charts/cryostat/README.md

```diff
@@ -32,6 +32,19 @@ A Helm chart for deploying [Cryostat](https://cryostat.io/) on Kubernetes and Op
 | `core.databaseSecretName`                      | Name of the secret to extract password for credentials database.                                                                                                                                                                                                                                               | `""`                        |
 
+
+### Storage Container
+
+| Name                       | Description                                                                                                                                                                                                                                                                                                   | Value                               |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| `Configuration`            | for Cryostat's object storage provider                                                                                                                                                                                                                                                                        |                                     |
+| `storage.image.repository` | Repository for the storage container image                                                                                                                                                                                                                                                                    | `quay.io/cryostat/cryostat-storage` |
+| `storage.image.pullPolicy` | Image pull policy for the storage container image                                                                                                                                                                                                                                                             | `Always`                            |
+| `storage.image.tag`        | Tag for the storage container image                                                                                                                                                                                                                                                                           | `2023-12-19`                        |
```

Comment:
I think the README/schema need to be regenerated after changing the default to `latest`.